package.json

```diff
@@ -2,12 +2,13 @@
   "name": "libxmljs",
   "author": "Marco Rogers",
   "contributors": [
-    "Jeff Smick"
+    "Jeff Smick",
+    "Martin von Gagern <Martin.vGagern@gmx.net>"
```

Comment:
This package is part of the npm ecosystem, which decided to employ the `package.json` file as required metadata. So people may expect to find the list of contributors there. The `AUTHORS` file is described as an alternative, and I think that npm will incorporate the information from that into the package description when publishing.

Not having a list of contributors in there at all *may* suggest to some to go looking for that information elsewhere. But having an author and/or some contributors listed makes it look authoritative, thus preventing people from further investigation. So removing all that info from the `package.json` *might* be an option, but it would probably confuse npm and there would still be that copyright holder mentioned in the `LICENSE` file, so I'm not too fond of this approach.

Not everyone is using git and GitHub. So to some people, the files shipped with the released version of the package, as downloaded from npm, are the canonical place to look for such information. One day, one of them may be deciding whether or not to hire me for something…

I think that ideally everyone who contributed significant amounts of work should be mentioned. But I didn't see it as my role to do that in a pull request here, so I only added my own name. In the fork I mantained, I [had added](https://github.com/gagern/libxmljs/commit/46e6d99aa32cd856a7d42f1831cc317f9d2e3f85#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) @defunctzombie, too. I probably should have added some others, too, so I don't claim to properly maintain that list either.

Probably the easiest way to not worry about whom to add is by adding every author git knows about. E.g. `git log --format='%aN <%aE>' | sort | uniq > AUTHORS` and edit to remove different email addresses for the same person. Also remove author and contributors from `package.json` since otherwise those take precedence.